x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.ts

```diff
@@ -22,7 +23,15 @@ export function getOpsStatsCollector(
   metrics$: Observable<OpsMetrics>
 ) {
   let lastMetrics: MonitoringOpsMetrics | null = null;
-  metrics$.subscribe(metrics => {
+  metrics$.subscribe(_metrics => {
+    const metrics = cloneDeep(_metrics);
+    // Ensure we only include the same data that Metricbeat collection would get
+    metrics.process = omit(metrics.process, 'pid');
+    set(metrics, 'response_times', {
+      average: metrics.response_times.avg_in_millis,
+      max: metrics.response_times.max_in_millis,
+    });
```

Comment:
NIT: can't this be replaced by just `metrics.response_times = ...` ?

Reply:
Done